tests/nnfw_api/src/GenModelTests/one_op_trains/FullyConnected.test.cc

```diff
@@ -33,9 +33,12 @@ TEST_F(GenModelTrain, OneOp_FullyConnected)
 
   float learning_rate = 0.01f;
   int32_t batch_size = 1;
-  cgen.addTrainInfo({circle::Optimizer::Optimizer_SGD, learning_rate,
+  cgen.addTrainInfo({circle::Optimizer::Optimizer_SGD,
+                     learning_rate,
                      circle::LossFn::LossFn_MEAN_SQUARED_ERROR,
-                     circle::LossReductionType::LossReductionType_SumOverBatchSize, batch_size});
+                     circle::LossReductionType::LossReductionType_SumOverBatchSize,
+                     batch_size,
+                     {0}});
```

Comment:
I have a question.
Can it be guaranteed that the op sequence number added to `cgen` above and the number to apply to here always are the same?

Reply:
I see your point. It can be dangerous. My proposition is a new method `CirclePlusGen::markAllOpsAsTrainable`